oonib.conf.example

```diff
@@ -30,12 +30,13 @@ main:
     tor_datadir: null
 
     bouncer_endpoints:
-    - {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
+    #- {type: tls, port: 10443, cert: "private/ssl-key-and-cert.pem"}
     - {type: tcp, port: 10080}
-    - {type: onion, hsdir: bouncer}
+    - {type: onion, hsdir: /tmp/bouncer}
```

Comment:
I commented it out because I am thinking that it would be good that by default when you start oonib using vagrant it "just works" instead of having to run some extra scripts.

Since it's in the example people should be able to figure out how to make it work, but I wouldn't want to degrade the experience with unattended setup.